### DISCUSSION TITLE
博主有联系方式吗？想谈合作~

### DISCUSSION THREAD

**Discussion by sssssshift** [hongtaoh/hongtaoh.github.io#132] [Announcements]

**Comment by hongtaoh** [comment]
我的联系方式在我的网站上有。

**Comment by sssssshift** [comment]
可以加您的微信进一步沟通吗？

**Comment by sssssshift** [comment]
近一步

**Comment by hongtaoh** [comment]
我想先看看是合作干啥。我们先邮件联系吧。

**Comment by sssssshift** [comment]
那我给您发个邮件吧 方便发过来吗 在您的网站上没看到联系方式

**Comment by hongtaoh** [comment]
你好，我的联系方式在 「中文」- 「关于」页面

**Comment by sssssshift** [comment]
好滴 谢谢您



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
23/03/31
14:37:34

Hongtao Hao ***@***.***> 于2023年3月31日周五 07:41写道：
